src/elfpy/pricing_models/base.py

```diff
@@ -312,10 +314,6 @@ def _calc_spot_price_from_reserves_high_precision(
         Decimal
             The spot price of principal tokens.
         """
-        assert market_state.share_reserves > 0, (
```

Comment:
do we not want this any more, was this moved somewhere?

Reply:
ya in the part below in types.py. you don't see this in the diff, but apply_delta() is used for updating market reserves, so it checks all 6 reserve values.